settings.py

```diff
@@ -0,0 +1,224 @@
+import os
+gettext = lambda s: s
+DATA_DIR = os.path.dirname(os.path.dirname(__file__))
+"""
+Django settings for preview project.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.7/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.7/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+BASE_DIR = os.path.dirname(os.path.dirname(__file__))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.7/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '=3x!$%2s_5jht5=i-j#9-bfpcup0sn*2&_nelyoql&+0p5itgs'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+TEMPLATE_DEBUG = True
+
+ALLOWED_HOSTS = []
+
+
+# Application definition
+
+
+
+
+
+ROOT_URLCONF = 'preview.urls'
+
+WSGI_APPLICATION = 'preview.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/1.7/ref/settings/#databases
+
+
+
+# Internationalization
+# https://docs.djangoproject.com/en/1.7/topics/i18n/
+
+LANGUAGE_CODE = 'en'
+
+TIME_ZONE = 'Etc/UTC'
+
+USE_I18N = True
+
+USE_L10N = True
+
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/1.7/howto/static-files/
+
+STATIC_URL = '/static/'
+MEDIA_URL = '/media/'
+MEDIA_ROOT = os.path.join(DATA_DIR, 'media')
+STATIC_ROOT = os.path.join(DATA_DIR, 'static')
+
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'preview', 'static'),
+)
+SITE_ID = 1
+
```

Comment:
if you plan to use ``bootstrap3`` boilerplate which is a supported and most recent for essential addons you need yo specify ``ALDRYN_BOILERPLATE_NAME = 'bootstrap3'`` otherwise you might end up with errors for some of the addons which do not have ``'legacy'`` boilerplates.
Usually I'm adding this setting before ``STATICFILES_FINDERS``